packages/manager/apps/catalog/src/pages/index.tsx

```diff
@@ -27,6 +30,15 @@ export default function CatalogRevamp() {
     searchText,
   });
 
+  useEffect(() => {
+    const params = new URLSearchParams();
+    if (searchText) params.append('q', searchText);
+    if (categories.length > 0)
+      params.append('categories', categories.join(','));
+    if (universes.length > 0) params.append('universes', universes.join(','));
+    navigate({ search: params.toString() });
```

Comment:
I feel that we should put this in a function and the content of the useEffect of the Filters.tsx file because, if for any reason, we change the parameter or we add new parameters, it will be easier to refactor if the functions are next to each other.
They will be easy to unit test, also.

Reply:
Done, getSearchUrlFromFilterParams created ✅

Reply:
Ok nice
Can you move it to the same file as the other function that extracts params from url ?
Like an helper or utils file